monkey/monkey_island/cc/ui/src/components/ui-components/Select.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {InputLabel, FormControl, Select, MenuItem, SelectChangeEvent} from '@mui/material';
+
+
+type Option = {
+  label: string,
+  value: string,
+}
+
+type SelectProps = {
```

Comment:
What about the variant?

Reply:
Like what? I don't think select has any variants

Reply:
https://mui.com/material-ui/api/select/#Select-prop-variant